ERCS/erc-zk-proof-verification-interface.md

```diff
@@ -0,0 +1,350 @@
+---
+eip: <to be assigned>
+title: Standard ZK Proof Verification Interface for Smart Contracts
```

Comment:
```suggestion
title: ZK Proof Verification
```
Standard and smart contracts are superfluous in the title for an ERC.  Note: I am not an ERC editor.